tools/detect_duplicated_commentids.php

```diff
@@ -102,7 +102,7 @@
             $candid = $candidate['CandID'];
             $pscid  = $candidate['PSCID'];
             foreach ($cohortids as $cohortid) {
-                $session_info = $DB->pselectRow(
+                $session_info = $DB->pselect(
```

Comment:
$session_info = $DB->pselectRow(
                    "SELECT DISTINCT s.Visit_label,s.ID from session s
                    JOIN candidate c on (c.candid=s.candid)
                    JOIN flag f on (f.sessionid=s.id)
                    WHERE s.candID = :cid AND f.test_name = :fname AND
                    s.cohortid = :subid 
                    limit =1",
                    [
                        'cid'   => $candid,
                        'fname' => $instrument,
                        'subid' => $cohortid['cohortid'],
                    ]
                );